servers/republik/modules/crowdfundings/lib/scheduler/owners.js

```diff
@@ -72,16 +73,15 @@ const getBuckets = async ({ now }, { pgdb }) => {
       m."userId" != :PARKING_USER_ID
       AND m.active = true
       AND m.renew = true
```

Comment:
Please indicate with a big fat comment, that this only hold when a user only has one active membership